src/Kunstmaan/FixturesBundle/Builder/MediaBuilder.php

```diff
@@ -18,17 +19,21 @@ class MediaBuilder implements BuilderInterface
     private $fileHandler;
 
     /**
-     * @var MimeTypeGuesserInterface
+     * @var MimeTypes|MimeTypeGuesser
      */
     private $mimeTypeGuesser;
 
     private $folder;
 
-    public function __construct(EntityManager $em, FileHandler $fileHandler, MimeTypeGuesserFactoryInterface $mimeTypeGuesserFactory)
+    public function __construct(EntityManager $em, FileHandler $fileHandler, $mimeTypeGuesser)
     {
         $this->em = $em;
         $this->fileHandler = $fileHandler;
-        $this->mimeTypeGuesser = $mimeTypeGuesserFactory->get();
+        $this->mimeTypeGuesser = $mimeTypeGuesser;
+        if ($mimeTypeGuesser instanceof MimeTypeGuesserFactoryInterface) {
+            $this->mimeTypeGuesser = $mimeTypeGuesser->get();
+            @trigger_error('Passing the "@kunstmaan_media.mimetype_guesser.factory" service as second argument is deprecated since KunstmaanMediaBundle 5.6 and will be replaced by the "@mime_types" service in KunstmaanMediaBundle 6.0. Inject the correct service instead.', E_USER_DEPRECATED);
```

Comment:
I would reword this to 
```suggestion
            @trigger_error('Passing a service instance of "\Kunstmaan\MediaBundle\Helper\MimeTypeGuesserFactoryInterface" as the second argument is deprecated since KunstmaanMediaBundle 5.6 and will be replaced by the "@mime_types" service in KunstmaanMediaBundle 6.0. Inject the correct service instead.', E_USER_DEPRECATED);
```